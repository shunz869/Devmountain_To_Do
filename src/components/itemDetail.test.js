import React from 'react';
import ItemDetail from './itemDetail';
import renderer from 'react-test-renderer';

test('ItemDetail renders correctly', ()=>{
  const component = renderer.create(<ItemDetail item={{
    id:1,
    title:'abc'
  }} />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})