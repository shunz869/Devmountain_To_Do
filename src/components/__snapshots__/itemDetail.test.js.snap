// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemDetail renders correctly 1`] = `
<form
  style={
    Object {
      "marginTop": 50,
    }
  }
>
  <div
    className="form-group"
  >
    <label>
      Task
    </label>
    <input
      className="form-control"
      placeholder="abc"
      type="text"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Description
    </label>
    <input
      className="form-control"
      type="text"
    />
  </div>
  <div
    className="btn-group"
  >
    <button
      className="btn btn-primary"
      onClick={[Function]}
      style={
        Object {
          "marginRight": 5,
        }
      }
    >
      Save
    </button>
    <button
      className="btn btn-secondary"
      onClick={[Function]}
      style={
        Object {
          "marginRight": 5,
        }
      }
    >
      Cancel
    </button>
  </div>
</form>
`;
